src/utils/generator.ts

```diff
@@ -45,6 +48,9 @@ export default class Generator {
       }
     }
 
+    // https://developer.chrome.com/articles/new-headless/
+    puppeteerOptions.headless = 'new'
```

Comment:
I've done several tests locally, and with this option, PDFs are generated *much* faster.